dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/DdAndroidGradlePlugin.kt

```diff
@@ -94,6 +94,11 @@ class DdAndroidGradlePlugin @Inject constructor(
         } else {
             LOGGER.info("Minifying disabled for variant ${variant.name}, no upload task created")
         }
+        configureSymbolUploadTask(
```

Comment:
Yes, I saw that. I mean maybe it makes sense for the `register` method to return non-null value always, and do the check for the native build providers before the `register` call, but then I guess it will be harder to unit-test it.